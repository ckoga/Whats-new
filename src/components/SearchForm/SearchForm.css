input,
button {
  height: 60px;
}

input {
  font-size: 1em;
  border: none;
  border-bottom: 1px solid white;
  color: white;
  background-color: #141527;
  width: 400px;
}

button {
  border: none;
  background-color: #B56BF2;
  color: white;
  font-size: 1em;
  width: 150px;
  border-radius: 30px;
  cursor: pointer;
}