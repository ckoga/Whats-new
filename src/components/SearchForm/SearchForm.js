import React from 'react';
import './SearchForm.css';

// SEARCHFORM COMPONENT CODE GOES HERE
const SearchForm = (props) => {
  return (
    <form>
      <input className="input" placeholder="Search News"></input>
      <button className="search-button" onClick={(event) => props.searchNews(event)} type="button">Search</button>
    </form>
  )
}

export default SearchForm;