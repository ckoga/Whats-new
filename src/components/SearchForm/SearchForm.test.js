import React from 'react';
import { shallow } from 'enzyme';
import SearchForm from './SearchForm';

describe('SearchForm', () => {
  it('should be able to render SearchForm to the DOM', () => {
    const wrapper = shallow(<SearchForm 
      searchNews={jest.fn()}
    />)
    
    expect(wrapper).toMatchSnapshot();
  })
})