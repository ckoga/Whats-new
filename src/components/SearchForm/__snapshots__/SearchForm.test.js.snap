// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchForm should be able to render SearchForm to the DOM 1`] = `
<form>
  <input
    className="input"
    placeholder="Search News"
  />
  <button
    className="search-button"
    onClick={[Function]}
    type="button"
  >
    Search
  </button>
</form>
`;
